oat-sa/extension-tao-testqti
# Feature/BSA-255/add translations for itemThemeSwitcher plugin

Related to: https://oat-sa.atlassian.net/browse/BSA-255
https://github.com/oat-sa/extension-tao-colorcontrast/pull/15

Added German, French, Dutch translations itemThemeSwitcher plugin.

How to test:
- create item
- change language to German/French/Dutch in settings
- run item Preview
- check Contrast Plugin
![image](https://user-images.githubusercontent.com/25976342/94906475-34fd5800-04a7-11eb-8569-c80259e345ae.png)
